src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1268,14 +1268,16 @@ static MethodSymbol getTopLevelMethod(MethodSymbol method)
             }
         }
 
+#nullable enable
         protected override int GetOrCreateSlot(Symbol symbol, int containingSlot = 0, bool forceSlotEvenIfEmpty = false)
         {
 
-            if (containingSlot > 0 && !IsSlotMember(containingSlot, symbol))
+            if ((containingSlot > 0 && !IsSlotMember(containingSlot, symbol)) || symbol is null)
```

Comment:
>|| symbol is null [](start = 78, length = 17)

We should check `symbol is null` higher in the callstack where we know what the `symbol` represents. (Sorry, I didn't realize the caller of the base method was the `override` when I suggested moving the check previously.)

Reply:
Moved the check to the place where the null object is in Commit 8.

---
In reply to: [447985704](https://github.com/dotnet/roslyn/pull/45563#discussion_r447985704) [](ancestors = 447985704)